apps/react-app/src/providers/mac.provider.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { MacContext } from "./mac.context";
+
+interface Props {
+  children: React.ReactNode;
+}
+
+export const MacProvider: React.FC<Props> = ({ children }) => {
```

Comment:
_Mac_ can confuse other developers, we need to try to be consistent, in some parts we are talking about ManfredJsonContent, here we describe it as MAC (name of the repo that hold the ManfredJSOn definition).

There's no silverbullet, we could discuss here:
 - Cool MAC is more generic and in the future we may be introduce more info than just only the (JSON), e.g. export to Word, or PDF, o HTML...).
 - JsonManfred is more accurate.

Let's elaborate a little bit more about this:
  - We are just setting a value on a wizard like process (right now only two steps, maybe in the future it will contain more steps).
  - One of the fields is the JSONManfred.

A quick and valid solution could be to store just the JSON string and create a context specifically for it, if we need in the future to go for that, well let's refactor (YAGNI, you ain't gonna need it, why implementing that now).

Another solution, could be to name it something like _UserChoiceProvider_ and inside add a field for the JSONContext, in the future we could just add field to the _UserChoiceModel_, like e.g. expor to Word / HTML/ PDF...

If you all agree we could rename this to _userChoice_ and in the model include a field that explicitly reffers to the JSONManfred: manfredJson : string